azplugins/test-py/test_variant_sphere_area.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2018-2019, Michael P. Howard
+# This file is part of the azplugins project, released under the Modified BSD License.
+
+# Maintainer: mphoward
+
+import hoomd
+from hoomd import md
+hoomd.context.initialize()
+try:
+    from hoomd import azplugins
+except ImportError:
+    import azplugins
+import unittest
+
+class variant_sphere_area_tests(unittest.TestCase):
+    def setUp(self):
+        hoomd.init.read_snapshot(hoomd.data.make_snapshot(N=0, box=hoomd.data.boxdim(L=20.)))
+
+    def test_create(self):
+        """Test creation of sphere objects."""
+        # floats
+        s = azplugins.variant.sphere_area(R0=10., alpha=0.5)
+        self.assertAlmostEqual(s.R0, 10.)
+        self.assertAlmostEqual(s.alpha, 0.5)
+        self.assertTrue(s.cpp_variant is not None)
+
+        # ints
+        azplugins.variant.sphere_area(R0=10, alpha=1)
```

Comment:
Is the test incomplete? Do we not want to test whether the values have been correctly set?

Reply:
Good to check, but no, I only added this test to ensure that the implicit typecast from int to float worked correctly in the constructor (it would raise an exception otherwise). Sometimes this fails with pybind11.